### PR TITLE
Describe how to add UKI splash screen

### PR DESCRIPTION
**Pull request by bencorrado**
Docs support for https://github.com/kairos-io/enki/pull/209 and closes https://github.com/kairos-io/kairos/issues/2899

### CODE REVIEW

**Comment by bencorrado** [comment]
@mauromorales can you add #hacktoberfest when you get a chance?

**Comment by mauromorales** [comment]
@bencorrado repo has it, so it should be properly picked. Let me know if it doesn't once merged

**Review by mauromorales — APPROVED** [review]
Nice! Thanks for the PR 🙌

File: content/en/docs/Installation/trustedboot.md [hunk 1]
```diff
@@ -176,6 +176,20 @@ efi /EFI/kairos/active.efi
 title My Awesome OS
 ```
 
+You can use a custom boot splash screen `--splash` flag to enki.
```
Comment (Itxaka) [inline content/en/docs/Installation/trustedboot.md:179]:
```suggestion
You can use a custom boot splash screen by specifying the  `--splash` flag when calling enki.
```

**Review by Itxaka — APPROVED** [review]

**Pull request merged**